src/result.js

```diff
@@ -457,11 +457,12 @@ var ResultList = CreateClass({
         var job = (prof.job == undefined) ? Jobs["none"].name : Jobs[prof.job].name;
         var charaInfoStr = intl.translate("ジータさん", locale) + "(" + intl.translate(job, locale) + ") HP";
         if (prof.remainHP != undefined) {
-            charaInfoStr += (parseInt(prof.remainHP) < parseInt(prof.hp)) ? prof.remainHP : prof.hp
+            charaInfoStr += (parseInt(prof.remainHP) < parseInt(prof.hp)) ? (prof.remainHP == 0 ? "1" : prof.remainHP) : (prof.hp == 0 ? "1" : prof.hp);
```

Comment:
it's not good style to nest ternary operator "? :"
https://eslint.org/docs/rules/no-nested-ternary

```js
Math.min(parseInt(prof.remainHP), parseInt(prof.hp)) || "1"
```

Note: this code assume HP is not negative numbers, when Math.min() returned 0 then "1"

Reply:
I understand. done.